ansible/roles/etcd/tasks/install.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install etcd
+  action: "{{ ansible_pkg_mgr }}"
+  args:
+        name: etcd
+        state: latest
+  notify:
+    - restart etcd
+  when: not is_atomic and ansible_distribution != "Ubuntu" and not is_coreos
```

Comment:
what got broken? ``etcd_source_type`` is not defined on CentOS? If so, please open a PR for it and reference it here. This needs to be fixed. I can imagine other values for ``etcd_source_type`` such as installation of the packages from the latest build in distribution builder not yet pushed to stable (e.g. ``etcd_source_type == "distribution_builder"``). On Fedora it makes sense to install etcd from rawhide build. I have already use case for this for docker.